cpu/native/Makefile.dep

```diff
@@ -41,11 +41,8 @@ ifneq (,$(filter socket_zep,$(USEMODULE)))
 endif
 
 ifneq (,$(filter libc_gettimeofday,$(USEMODULE)))
-  USEMODULE += xtimer
-  ifneq (,$(filter ztimer_xtimer_compat,$(USEMODULE)))
-    # requires 64bit timestamps
-    USEMODULE += ztimer64_xtimer_compat
-  endif
+  USEMODULE += ztimer
+  USEMODULE += ztimer_usec
```

Comment:
```suggestion
  USEMODULE += ztimer64_usec
```

Reply:
Maybe it's my phone's diff view, but it still shows `ztimer_msec` vs. `ztimer64_msec` (also for Kconfig)

Reply:
Ya, makes sense...